canonical-web-and-design/maas-ui
# Use REACT_APP_* env vars for configuration.

## Done
* Migrate configuration from `config.js` to [react app env vars](https://facebook.github.io/create-react-app/docs/adding-custom-environment-variables).

## QA
* Run the app, ensure it still works (can connect to MAAS).
* If you'd like, `console.log(process.env)` somewhere at the top of the app to ensure values are set correctly.
